http/0.3.7/modules/ROOT/pages/http-connector-reference.adoc

```diff
@@ -386,7 +386,7 @@ HTTP global configuration.
 
 === Attributes of config
 
-`useTransportForUris`: Backwards Compatabilty Flag: Since Mule 3.6, default HTTP URIs are resolved with the new HTTP connector (for example when using MuleClient). If set to true, this behavior is changed so that the HTTP transport is used.
+`useTransportForUris`: Backwards Compatability Flag: Since Mule 3.6, default HTTP URIs are resolved with the new HTTP connector (for example when using MuleClient). If set to true, this behavior is changed so that the HTTP transport is used.
```

Comment:
```suggestion
`useTransportForUris`: Backwards Compatibility Flag: Since Mule 3.6, default HTTP URIs are resolved with the new HTTP connector (for example when using MuleClient). If set to true, this behavior is changed so that the HTTP transport is used.
```